ballista/client/tests/context_checks.rs

```diff
@@ -240,12 +240,12 @@ mod supported {
             .await?;
 
         let expected = [
-            "+---------------------------------------+----------+",
-            "| name                                  | value    |",
-            "+---------------------------------------+----------+",
-            "| ballista.grpc_client_max_message_size | 16777216 |",
-            "| ballista.job.name                     |          |",
-            "+---------------------------------------+----------+",
+            "+-------------------------------------------------------+-------+",
+            "| name                                                  | value |",
+            "+-------------------------------------------------------+-------+",
```

Comment:
can we please update query to return ` ballista.job.name` only. so we don't change the test every time we add new option.